src/server/JasmineGraphServer.cpp

```diff
@@ -1994,31 +1994,53 @@ void JasmineGraphServer::initiateOrgCommunication(std::string graphID, std::stri
 
     Utils utils;
     int fl_clients = stoi(utils.getJasmineGraphProperty("org.jasminegraph.fl_clients"));
-        std::string flagPath = utils.getJasmineGraphProperty("org.jasminegraph.fl.flag.file");
-    int threadLimit = fl_clients + 1;
+    int orgs_count = stoi(utils.getJasmineGraphProperty("org.jasminegraph.fl.num.orgs"));
+    std::string flagPath = utils.getJasmineGraphProperty("org.jasminegraph.fl.flag.file");
+    int threadLimit = fl_clients+1;
+    int org_thread_count = 0;
     std::thread *workerThreads = new std::thread[threadLimit];
 
     utils.editFlagOne(flagPath);
     Utils::worker workerInstance;
     vector<Utils::worker> workerVector = utils.getWorkerList(sqlite);
 
+    std::thread *trainThreads = new std::thread[orgs_count];
+
     if (utils.getJasmineGraphProperty("org.jasminegraph.fl.aggregator") == "true"){
    
         std::thread *orgAggThread = new std::thread[1];
         orgAggThread[0] = std::thread(initiateAggregator,"localhost",  stoi(workerVector[0].port), 
                                         stoi(workerVector[0].dataPort),trainingArgs,fl_clients, "1");
         orgAggThread[0].join();
+
+    }
+
+    std::ifstream file(utils.getJasmineGraphProperty("org.jasminegraph.fl.organization.file"));
```

Comment:
What will happen if this file is not specified in the `jasimegraph-server.properties config` file?

Reply:
Error handling added in 58678b1a64d640daa8361e0ff3620faf4205adb9

Reply:
Done